helix-front/server/controllers/helix.ts

```diff
@@ -51,6 +51,8 @@ export class HelixCtrl {
       request[method](options, (error, response, body) => {
         if (error) {
           res.status(500).send(error);
+        } else if (body?.error) {
+          res.status(500).send(body?.error);
```

Comment:
Handle the case where previously this error message would have been silently ignored:

![Screen Shot 2022-06-27 at 1 25 51 PM](https://user-images.githubusercontent.com/2119400/176029517-baf499e3-1daa-40ca-b819-d5cb2c6b1332.png)